requirements/base.in

```diff
@@ -0,0 +1,7 @@
+# Base requirements for using Flow-Control
+
+-c constraints.txt
+
+xblock                              # Base courseware component architecture
+xblock-utils                        # Provides utilities used by the Flow-Control XBlock
+edx-opaque-keys                     # Allows fetch scores from Courseware's ScoresClient
```

Comment:
Let's avoid the error that we had in eox-tagging, so change this with edx-opaque-keys[django] or remove this from here and move that to test.in

Reply:
Changed to edx-opaque-keys[django]. 
Added constraint <2.0.0